frame/support/test/tests/pallet_ui/deprecated_store_attr.stderr

```diff
@@ -0,0 +1,8 @@
+error: use of deprecated trait `pallet::Store`: Use of #[pallet::generate_store(pub(super) trait Store)] will be removed soon.
+               Check https://github.com/paritytech/substrate/pull/13535 for more details.
+  --> $DIR/deprecated_store_attr.rs:11:33
+   |
+11 |     #[pallet::generate_store(trait Store)]
+   |                                    ^^^^^
```

Comment:
@vivekvpandya What Basti meant was that the span here should be pointing to the entire attribute (i.e. `#[pallet::generate_store(trait Store)]`), and not just `Store`.

Reply:
Oh really interesting point

Reply:
Okay I see that magic happens at https://github.com/paritytech/substrate/commit/cb4f2491b00af7d7817f3a54209c26b20faa1f51#diff-73ebad034360f06fb4956cfe4cd26ddc26cb1ae6f1c4bb85dae098f7bca99f81R65

Reply:
Yes, I already had written this here: https://github.com/paritytech/substrate/pull/13535#issuecomment-1457814580

Ty for doing this @vivekvpandya